fido-device-onboard.spec

```diff
@@ -0,0 +1,127 @@
+%define dracutlibdir %{_prefix}/lib/dracut
+%bcond_without check
+%global __cargo_skip_build 0
+%global __cargo_is_lib() false
+%global forgeurl https://github.com/fedora-iot/fido-device-onboard-rs
+
+%forgemeta
+
+Name:           fido-device-onboard
+Version:        0.1.0
+Release:        1%{?dist}
+Summary:        An implementation of the FIDO Device Onboard Specification written in rust
+
+License:        BSD 3
+URL:            https://github.com/fedora-iot/fido-device-onboard-rs
+Source:         %{url}/archive/%{commit}/%{name}-rs-%{commit}.tar.gz
+%if 0%{?rhel} && !0%{?eln}
+Source1:        %{name}-rs-%{commit}-vendor.tar.gz
+%endif
+
+ExclusiveArch:  %{rust_arches}
+# RHBZ 1869980
+ExcludeArch:    s390x i686 %{power64}
+
+%if 0%{?rhel} && !0%{?eln}
+BuildRequires:  rust-toolset
+%else
+BuildRequires:  rust-packaging
+%endif
+BuildRequires: systemd
+
+%description
+%{summary}.
+
+%prep
+%autosetup -n %{name}-rs-%{commit} -p1
+%if 0%{?rhel} && !0%{?eln}
+%cargo_prep -V 1
```

Comment:
Is there a reason we need to -V 1 here and not elsewhere or the reverse?

Reply:
this is just for RHEL to use vendored deps